src/main/java/org/onebusaway/alexa/lib/ObaUserClient.java

```diff
@@ -35,18 +37,37 @@
 public class ObaUserClient {
     private static final int DEFAULT_SEARCH_RADIUS_METERS = 40000;
 
-    public ObaUserClient(String endpoint) {
-        log.debug("Instantiating ObaUserClient with endpoint " + endpoint);
+    public ObaUserClient(String obaBaseUrl) {
```

Comment:
You're passing into this method, as an argument, directly the return value of `getObaBaseUrl()`. That currently returns either String or null, leading to an occasional error or crash.